README.md

```diff
@@ -49,831 +49,17 @@ Unsupported parameters for ssh_config:
 AddressFamily, Tunnel, TunnelDevice, PermitLocalCommand, HashKnownHosts
 
 Unsupported parameters for sshd_config:
-KerberosOrLocalPasswd, KerberosTicketCleanup, KerberosGetAFSToken, TCPKeepAlive, ShowPatchLevel, MaxSessions, PermitTunnel
+KerberosOrLocalPasswd, KerberosTicketCleanup, KerberosGetAFSToken, TCPKeepAlive, ShowPatchLevel,
```

Comment:
Are these unsupported really as we have the custom param? Perhaps this area can be removed.

Reply:
I agree it, this piece of info seems a little bit lonely here, will remove it. Sure the provided default settings doesn't use them.

Also I have added that this module is based around OpenSSH (but can be used for any other as well).

Reply:
did you see that this is a list of incompatible parameters that should be used with Sun SSH implementations.

Reply:
I did not.. a `### SunSSH` subsection would be helpful there.